client/pages/verify.js

```diff
@@ -0,0 +1,218 @@
+/* eslint-disable no-console */
```

Comment:
remove this disable directive.

also never disable `no-console`. it's a warning that won't break your build for a reason. `console.log` should only ever be used in debugging and they need to be removed before merging to develop.

Reply:
Got it! Won't disable it again